### PR TITLE
Sw

### PR DESCRIPTION
**Pull request by lithin**
### Motivation

- #104 step int he right direction - if a user has loaded any page at least twice, they'll never see the dinosaur again. That is, if they us a hyper-modern browser :D

### Test plan

- in Chrome and Firefox: load any website honestly page at least twice; then go offline and test that it still loads
- check that all assets still load fine in all other browsers

### Pre-merge checklist

- [ ] Unit tests
- [x] Reviews
  - [x] Code review
  - [x] Design review N/A
- [ ] Manual testing
  - [x] Windows 7 IE 11 N/A
  - [x] Windows 10 Edge N/A
  - [x] Mac OS El Capitan Safari N/A
  - [x] Mac OS El Capitan Chrome
  - [x] Mac OS El Capitan Firefox (supports SW but doesn't work offline)
  - [x] iOS 9 Safari N/A
  - [ ] Android 6 Chrome (should work fully but can't test it via browserstack)
- [ ] Tester approved

### CODE REVIEW

File: bin/ci-deploy.sh [hunk 1]
```diff
@@ -34,6 +35,7 @@ deployMaster() {
   echo Copying assets to S3
   aws s3 sync ./dist/assets-honestly s3://$BUCKET_NAME/assets-honestly
   aws s3 sync ./dist/txt s3://$BUCKET_NAME
+  aws s3 cp ./dist/sw.js s3://$BUCKET_NAME/$COMMIT_REF
```
Comment (zoeabryant) [inline bin/ci-deploy.sh]: no ref

File: site/compiler/asset-paths.js [hunk 2]
```diff
@@ -10,3 +10,5 @@ const assets = assetsDigest[bundleName];
 
 export const cssPath = filePath(assets.css);
 export const jsPath = filePath(assets.js);
+
+export const HI = 'HI';
```
Comment (zoeabryant) [inline site/compiler/asset-paths.js]: gone

**Comment by lpil** [comment]
Does this work for other pages or just the index?

**Comment by lithin** [comment]
Works everywhere! :)

**Comment by asavin** [comment]
Since it is impossible to test it on preview app because of the SHA in the url, should we merge this, test on staging and see if it works?

**Comment by lpil** [comment]
It's not impossible, just import the path prefix the same way that the router does :)

**Pull request merged**